### ISSUE TITLE
Manage Idle Tabs In Aniwatch glitch

### ISSUE THREAD

**Issue by ImJonSolo** [Serraniel/AniwatchPlus#193]
if you enable manage idle tabs and disable it, itll still work if you have the aniwatch plus extension.

**Comment by kaffem** [comment]
Thats because we as well implemented a feature with the same functionality as the "manage idle tabs" by aniwatch.
There are two settings for AniwatchPlus:
- Autoplay after tab focused
- Autopause after tab unfocused

**Comment by ImJonSolo** [comment]
oh ty for explaining

**Comment by Serraniel** [comment]
Yes, we either must have overseen the option or they implemented this feature recently during the week we were preparing the release.
Will keep the issue open to discuss if we should remove our implementation then, as it (more or less) is useless now.


@ImJonSolo In general, after you change the plugin settings, you also have to press the reload button in your browser before the changes take effect. Pressing `F5` key won´t be enough because of the custom reload implemented by Aniwatch.

**Comment by ImJonSolo** [comment]
Ok, thank you for explaining this and nice work, very nice extension.
